### DISCUSSION TITLE
Problem with Latex editing

### DISCUSSION THREAD

**Discussion by A-tA-v** [JetBrains/ideavim#514] [Q&A]
Hi All, I have a problem editing Latex file when both Texify and IdeaVim plugins are installed:
after any text alteration IDE stops responding for dozens of seconds, CPU load 100%.
Editing is simply impossible, although Latex compilation is fine.

Also, the IDE tries to render formulas substituting Latex commands by graphical symbols.
I suspect, the rendering might be an issue.  In my ~/.ideavimrc the following stuff was added
to disable rendering:
```
let g:tex_conceal = ""
let g:tex_noindent_env = 'document\|verbatim\|lstlisting\|frame\|equation'
let g:vimtex_indent_ignored_envs = ['document', 'verbatim', 'lstlisting', 'frame', 'equation']
let g:vimtex_syntax_conceal = ""
```
but it does **not** help - rendering is still there. 

I am not sure which plugin (Texify or IdeaVim) makes the problem, however, I have experienced similar behaviour with plain VIM and managed to fix it by adding the above lines in ~/.vimrc. This time the remedy does not work.

System: OSX Monterey. 

I would appreciate any suggestion to solve the problem.

**Comment by PHPirates** [comment]
Cross-posted at https://github.com/Hannah-Sten/TeXiFy-IDEA/issues/2433
